### PR TITLE
Improve MapboxNavigation project structure.

### PR DESCRIPTION
**Pull request by MaximAlien**
### Description
This PR aims to slightly improve project structure in MapboxNavigation (UI part), remove unused code and imports.
Previously some entities were not grouped (e.g. feedback, instructions, speech etc), it made it harder to navigate throughout the code. 

### Screenshots or Gifs
Here's how new project structure looks like:
Screen Shot 2020-12-07 at 5 53 41 PM

### CODE REVIEW

File: MapboxNavigation/Cache.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@ public typealias CompletionHandler = () -> Void
 /**
  A cache consists of both in-memory and on-disk components, both of which can be reset.
  */
-@objc(MBBimodalCache)        
+@objc(MBBimodalCache)
```
Comment (1ec5) [inline MapboxNavigation/Cache.swift:8]: Any idea if we need this class to be exposed to the Objective-C runtime?

**Review by 1ec5 — APPROVED** [review]
Thanks for tidying up the project!

Finally, that space in “MapboxNavigation/CarPlaySearchController+&nbsp;CPSearchTemplateDelegate.swift” is gone – that has been bugging me for a long time but I keep forgetting to fix it.

File: MapboxNavigation/Cache.swift (hunk 1 above)
Comment (MaximAlien) [reply MapboxNavigation/Cache.swift:8]: Not sure, I think it should be safe to remove it.

Comment (MaximAlien) [reply MapboxNavigation/Cache.swift:8]: Turns out `MBBimodalCache` should be exposed. Will leave it. I've also updated indentation in MapboxNavigation target to match default settings.

**Pull request merged**